content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -8,8 +8,7 @@ weight: 30
 <!-- overview -->
 
 This page shows how to configure Pods so that they will be assigned particular
-Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about
-scheduling and evicting Pods.
+Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about evicting Pods.
```

Comment:
AIUI the particular flavor of eviction when QoS class might be relevant is specifically preemption, and we could mention that.
```suggestion
Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about
which Pods to {{<glossary_tooltip text="preempt" term_id="preemption" >}}.
```
```

Reply:
Actually no. It's node-pressure eviction.

Preemption needs to be rephrased